### PR TITLE
feat: Dockerfile improvements for production

### PR DESCRIPTION
**Pull request by mattbell87**
- PHP config path independant of version
- Dynamic UID/GID

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +232,18 @@ If you have custom modules you can mount them to the following directory:
 
 - **/var/www/html/modules/name-of-module**
 
+**PHP Configuration**
```
Comment (DerekCrannaford) [inline README.md:235]: This gives guidance for PHP.ini and nginx conf.d
 = significant for dev/test, also per strategies for moderating config injection if refactoring CDK to target container deployment onto EC2
 = GREAT!
 
However, in the wild, a prime use case for this is tuning page timeout for sites with known bottlenecks.
Unfortunately the paths cited here are insufficient for complete config (of timeout) vs permutations of nginx setup.
Also needed is influence over : /etc/php/8.1/fpm/pool.d/www.conf
Which the current CDK does not provision!

Can you add that path here also, to help preserve intelligence pending step-wise improvement of CDK capacity?

Comment (DerekCrannaford) [reply README.md:235]: (/etc/php/8.1/fpm/pool.d/www.conf ... noting, that is vanilla Ubuntu, Alpine path may resolve differently!)

File: .codepipeline/local-dev/50_xdebug.ini [hunk 2]
```diff
@@ -1,5 +0,0 @@
-; Uncomment and configure mode https://xdebug.org/docs/all_settings#xdebug.mode
```
Comment (DerekCrannaford) [inline .codepipeline/local-dev/50_xdebug.ini:1]: Not sure why you don't need this anymore, without apparent other changes to build strategy?

Comment (DerekCrannaford) [reply .codepipeline/local-dev/50_xdebug.ini:1]: OK - I think this is signed of by 'cmfive_dev_tools.sh'
Does that also mean "cmfive-boilerplate/.codepipeline/docker/start.sh" needs weeding out of its attempted XDebug settings?

File: README.md (hunk 1 above)
Comment (mattbell87) [reply README.md:235]: Added some more info

**Review by DerekCrannaford — APPROVED** [review]

**Pull request merged**